File: Source/Fantasy.Jobs/Try.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using Fantasy.XSerialization;

namespace Fantasy.Jobs
{
    [Instruction]
    [XSerializable("try", NamespaceUri = Consts.XNamespaceURI)]  
    internal class Try : Sequence, IConditionalObject 
    {
        public override void Execute()
        {
            base.ExecuteSequence();
        }


        [XAttribute("condition")]
        private string _condition = null;
        string IConditionalObject.Condition
        {
            get
            {
                return this._condition;
            }
            set
            {
                this._condition = value;
            }
        }
       
    }
}

File: branches/Stheno/AutoConverted/Fantasy.Jobs/Try.java
package Fantasy.Jobs;

import Fantasy.XSerialization.*;

//C# TO JAVA CONVERTER TODO TASK: Java annotations will not correspond to .NET attributes:
//[Instruction, XSerializable("try", NamespaceUri = Consts.XNamespaceURI)]
public class Try extends Sequence implements IConditionalObject
{
	@Override
	public void Execute()
	{
		super.ExecuteSequence();
	}


//C# TO JAVA CONVERTER TODO TASK: Java annotations will not correspond to .NET attributes:
	//[XAttribute("condition")]
	private String _condition = null;
	private String getCondition()
	{
		return this._condition;
	}
	private void setCondition(String value)
	{
		this._condition = value;
	}

}